tracing-example/src/main/java/org/jboss/resteasy/tracing/examples/sample.json

```diff
@@ -0,0 +1,196 @@
+{
```

Comment:
What's the use of this sample.json file?

Reply:
recycled